index.html

```diff
@@ -12185,6 +12186,68 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="property" id="aria-repeated">
+			<pdef>aria-repeated</pdef>
+			<div class="property-description">
+				<p>Indicates content is repeated for visual purposes.</p>
+				<p>The <code>aria-repeated</code> helps indicate that content is repeated, and that presenting the repeated content to assistive technology users could produce underesirable redundancy in the output. Unlike <pref>aria-hidden</pref>, <code>aria-repeated</code> content may be presented by screen readers to users when requested by the user.</p>
```

Comment:
> content **may** be presented by screen readers 

If it's a normative "may" (which is what I think you meant?) then it needs to be MAY.
If not, then please change it to something like:
> content can be presented by screen readers